### ISSUE TITLE
Wind speed update interval different to rest

### ISSUE THREAD

**Issue by lanky8804** [siku2/hass-weatherlink#33]
Hi
Is there any way to implement a new user input variable so the wind speed can be updated at a different time interval to the other sensors?
Many thanks

**Comment by siku2** [comment]
Yes, that's technically possible, but to be fully transparent, I don't have any plans to implement this myself. Happy to accept contributions though.

The wind speed is part of the live broadcast anyway, so it should already update about once a second.
